src/epplibwrapper/tests/common.py

```diff
@@ -0,0 +1,51 @@
+import os
+import logging
+
+from contextlib import contextmanager
+
+
+def get_handlers():
```

Comment:
If these aren't directly copied over (i.e. you have a good grasp of how they function), I would recommend adding some code comments inside here. If this is copied over from the old code though, no worries on that

Reply:
Copied over.